src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/tests/test_sf_commands.py

```diff
@@ -0,0 +1,12 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import unittest
+import azure.cli.command_modules.servicefabric.custom
+from azure.cli.testsdk import ScenarioTest
+
+class ServiceFabricTest(ScenarioTest):
+    def test_list_storage_account(self):
+        cluster_list = self.cmd('az sf cluster list').get_output_in_json()
+        assert len(cluster_list) > 0
```

Comment:
Not only should you cover all of your new commands, you should exercise as many of your parameters as possible. There are many parameter registrations that have nothing more than `help='Text..'`.  Usually more kwargs are necessary to support relevant scenarios.